### PR TITLE
Feature/trip methods tests

### LINKED ISSUE #5: Write out class methods and passing tests
Write out tests/methods for classes

### LINKED ISSUE #27: Trip methods
Write methods and create passing tests

### PR DESCRIPTION
**Pull request by e-spitz**
## Description

Add methods to Trip class, one to find cost of trip based off tripID and destinationID, the other to create timestamps of start date of trip and end date of trip based off trip duration. Time stamps are in milliseconds based off that date.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] run test
- [x] terminal

# Checklist:

- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/Trip.js [hunk 1]
```diff
@@ -8,7 +8,27 @@ class Trip {
...
+   this.startDateTimeStamp = startTime;
+   this.endDateTimeStamp = endTime;
+  }
+
+  calculateTripCost(destinations) {
+    const matchedDestination = destinations.find(dest => dest.id === this.destinationID);
+    const lodgingTotal = matchedDestination.estimatedLodgingCostPerDay * this.duration;
+    const flightTotal = matchedDestination.estimatedFlightCostPerPerson * this.travelers;
+    const tripTotal = ((lodgingTotal + flightTotal) * 1.1);
+    this.tripCost = Math.round(tripTotal);
```
Comment (Meekb) [inline src/Trip.js]: You could replace Math.round:

`~~(tripTotal)`

**Review by Meekb — APPROVED** [review]
Looks good! Added a line comment on a refactor opportunity.

File: src/Trip.js (hunk 1 above)
Comment (e-spitz) [reply src/Trip.js]: Ohhh hell yea. I'm going to give that a try! Thanks meekb.

**Comment by e-spitz** [comment]
Add the ~~ and also added a price per person method =)

**Pull request merged**